contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -310,6 +316,8 @@ contract RaidenMicroTransferChannels {
     /// @param _receiver_address The address that receives tokens.
     /// @param _deposit The amount of tokens that the sender escrows.
     function createChannelPrivate(address _sender_address, address _receiver_address, uint192 _deposit) private {
+        require(_deposit <= channel_deposit_bugbounty_limit);
```

Comment:
Didn't look at it thoroughly, but doesn't this still allow you to top up a channel with as many tokens as you want?

Reply:
Ah, nevermind. The 'updateInternalBalanceStructs' take care of this.

Reply:
there is another check at topup